launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -47,6 +47,8 @@
   <!-- Radar parameters -->
   <arg name="input/radar" default="/sensing/radar/detected_objects"/>
   <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_lanelet_filter.param.yaml"/>
+  <arg name="use_near_radar_fusion" default="false"/>
+  <arg name="far_object_merger_sync_queue_size" default="20"/>
```

Comment:
[Q] Where do you use this argument `far_object_merger_sync_queue_size` ?

Reply:
Sorry, I missed moving it.
Fixed at [dd2ed5a](https://github.com/autowarefoundation/autoware.universe/pull/5166/commits/dd2ed5a50dff71f2fdd553d26f7b4cece12f4b4d)